examples/data-objects/inventory-app/webpack.config.cjs

```diff
@@ -38,26 +36,20 @@ module.exports = (env) => {
 			output: {
 				filename: "[name].bundle.js",
 				path: path.resolve(__dirname, "dist"),
-				library: "[name]",
-				// https://github.com/webpack/webpack/issues/5767
-				// https://github.com/webpack/webpack/issues/7939
-				devtoolNamespace: "fluid-experimental/tree-api",
-				libraryTarget: "umd",
+				library: { name: "[name]", type: "umd" },
 			},
-			plugins: [
-				new webpack.ProvidePlugin({
-					process: "process/browser",
-				}),
-			],
-			// This impacts which files are watched by the dev server (and likely by webpack if watch is true).
-			// This should be configurable under devServer.static.watch
-			// (see https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md) but that does not seem to work.
-			// The CLI options for disabling watching don't seem to work either, so this may be a symptom of using webpack4 with the newer webpack-cli and webpack-dev-server.
```

Comment:
I still don't have a great understanding of whats going on here, but I think there is a separate config for static file content watching which the v5 migration guide incorrectly pointed to (or I just misunderstood it at the time). Regardless this API is documented and works fine, and we have finished the migration and that didn't impact this, so I see no value to these comments any more.

Reply:
Do we know why node_modules is ignored?  Seems we would want to watch changes to node_modules as well (particularly being a monorepo)?

Reply:
I think its mostly due to possible issues trying to watch hundreds of thousands of files. Oddly I don't have any issues if I remove that, but I do have issues with jest trying to watch files when running tests, so maybe the jest setup is just extra bad somehow? Sadly the file watching APIs are OS and even somewhat filesystem dependent, so its hard to know if removing this ignore will cause issues for some people.

Note that even if we did watch node modules, if changing other packages you would still need to build those packages explicitly since that wouldn't be watching their source.